.pre-commit-config.yaml

```diff
@@ -22,7 +22,7 @@ repos:
     hooks:
       - id: mypy
         language: system
-        entry: mypy
```

Comment:
not sure about that, I wanted the CI to be independent of poetry there should be another way to fix your problem without having to update the CI